app-emulation/fuse/fuse-1.5.7.ebuild

```diff
@@ -18,7 +18,7 @@ IUSE="alsa ao backend-fbcon backend-sdl backend-svga backend-X gpm joystick meml
 REQUIRED_USE="?? ( backend-fbcon backend-sdl backend-svga backend-X )"
 
 RDEPEND="
-	>=app-emulation/libspectrum-1.4.4
+	~app-emulation/libspectrum-1.4.4
```

Comment:
This change needs a revbump, 
https://devmanual.gentoo.org/general-concepts/ebuild-revisions/index.html

Reply:
It would be nice if this was enforced programmatically (repoman and/or github CI). It would save both my time and your time.

Reply:
pkgcheck scan --commits **can** do this. Please consider being more polite to reviewers. It's not always easy to convey tone via text, so it's sometimes better to err on the side of caution.